starter_code/public/stylesheets/style.css

```diff
@@ -0,0 +1,70 @@
+body {
+  padding: 50px;
+  font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
+}
+
+a {
+  color: #00b7ff;
+}
+
+#home {
+  width: 80vw;
```

Comment:
So I heard something interesting the other day that if it is possible to use % then it is better than vw because it apparently works better on every platform. So for instance it looks better if you use % rather than vw when you open the website on your phone, and then you flip the phone sideways.
I haven't been able to test if this is true myself, but wanted to share it in case you are interested;)